.github/workflows/test.yml

```diff
@@ -62,17 +62,18 @@ jobs:
         uses: codecov/codecov-action@v2
         with:
           files: xtgeocoverage.xml
-  integration:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v1
-      - uses: "./.github/actions/test_setup"
-        with:
-          python-version: 3.9
-      - name: Set up OPM
-        run: |
-          sudo apt install software-properties-common &&
-          sudo apt-add-repository ppa:opm/ppa &&
-          sudo apt install mpi-default-bin libopm-simulators-bin
-      - name: Integration test
-        run: HAS_OPM=1 python -m pytest -m requires_opm --disable-warnings -x --hypothesis-profile ci
+
+  # integration:
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - uses: actions/checkout@v1
+  #     - uses: "./.github/actions/test_setup"
+  #       with:
+  #         python-version: 3.9
+  #     - name: Set up OPM
+  #       run: |
+  #         sudo apt install software-properties-common &&
+  #         sudo apt-add-repository ppa:opm/ppa &&
+      #     sudo apt install mpi-default-bin libopm-simulators-bin
+      # - name: Integration test
+      #   run: HAS_OPM=1 python -m pytest -m requires_opm --disable-warnings -x --hypothesis-profile ci
```

Comment:
Personally I would delete this in a separate commit, and tag that commit in the issue to get it fixed instead of commenting out.